cardano-sync/src/Cardano/Sync/Util.hs

```diff
@@ -131,3 +132,14 @@ renderSlotList xs
   | length xs < 10 = textShow (map unSlotNo xs)
   | otherwise =
       mconcat [ "[", textShow (unSlotNo $ List.head xs), "..", textShow (unSlotNo $ List.last xs), "]" ]
+
+-- | This is required for testing or when disabling the metrics layer.
+emptyMetricsLayer :: MetricsLayer
+emptyMetricsLayer =
+    MetricsLayer
+        { mlSetNodeBlockHeight = \_ -> pure ()
```

Comment:
`const` ?

Reply:
This is much clearer to me than `const`.